spec/daru_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe 'Daru' do
+  it 'default error stream should be $stderr' do
+    expect(Daru.error_stream).to eq($stderr)
+  end
+
+  it 'should be able to set error stream to nil' do
+    Daru.error_stream = nil
+    expect(Daru.error_stream).to be_nil
+  end
```

Comment:
```ruby
it { expect { Daru.error_stream = nil }.to change(Daru, :error_stream).to(nil) }
```
Though, this test is completely unnecessary, as you are, in fact, testing Ruby's `attr_accessor` implementation (and the fact of existance of `error_stream=` is tested below anyways).